mapit_gb/controls/2017-10.py

```diff
@@ -0,0 +1,59 @@
+# A control file for importing Boundary-Line.
+
+from mapit.models import Area, Generation
+
+
+def code_version():
+    return 'gss'
+
+
+def check(name, type, country, geometry, ons_code, commit, **args):
+    """Should return True if this area is NEW, False if we should match"""
+
+    current = Generation.objects.current()
+
+    # May 2017 Boundary-Line gave Clydebank Central the wrong ID,
+    # return the relevant row if present
+    if type == 'UTW' and ons_code == 'S13003126':
+        return Area.objects.get(codes__type__code='gss', codes__code='S13003136')
```

Comment:
Should this be wrapped in a `try`/`except` like the ones below, or will it definitely always exist?

Reply:
If the area doesn't exist, it'll raise an exception to create a new area, which is what we want and what would happen anyway (as the same thing would happen when it tried to look up S13003126).